packages/cli/lib/cli.ts

```diff
@@ -314,20 +314,22 @@ export const tscWatch = async (debug = false) => {
     watcher.on('change', (filePath: string) => {
         if (filePath === nangoConfigFile) {
             // config file changed, re-compile each ts file
-            const integrationFiles = listFiles({});
+            const integrationFiles = listFilesToCompile();
             for (const file of integrationFiles) {
                 compileFile(file);
             }
             return;
         }
-        compileFile(listFile(filePath));
+        compileFile(getFileToCompile(filePath));
     });
 
-    const compiler = tsNode.create({
-        skipProject: true, // when installed locally we don't want ts-node to pick up the package tsconfig.json file
-        compilerOptions: JSON.parse(tsconfig).compilerOptions
-    });
     function compileFile(file: ListedFile) {
+        // This needs to be re-declared each time
```

Comment:
What was the issue here?

Reply:
If you put that here, file won't be recompiled after an other change. I guess there is some cache or whatever (unrelated to glob)

Reply:
gotcha, yeah, think I ran into this issue recently. Thanks.